src/components/Loader/Loader.tsx

```diff
@@ -0,0 +1,15 @@
+import classNames from 'classnames';
+import React from 'react';
+
+type Props = {
+  isActive: boolean,
+};
+
+export const Loader: React.FC<Props> = ({ isActive }) => {
+  return (
+    <div className={classNames('modal overlay', { 'is-active': isActive })}>
```

Comment:
Consider using a more descriptive class name instead of `modal overlay`. It is not clear what this class does. 🤔